Python.gitignore

```diff
@@ -47,6 +47,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 *.cover
+*.py,cover
```

Comment:
@thedrow is this the right extension? The `,` here seems like it'd be a huge problem on Windows, and I can't seem to find a mention of this in the documentation you linked to.

Reply:
That's the extension `coverage` generates.
Try it yourself.

Here's the code that says so BTW: https://github.com/nedbat/coveragepy/blob/502ecfc1767480ceefd0745d258a945163b61024/coverage/annotate.py#L21

Reply:
TIL: this actually works

<img width="465" src="https://user-images.githubusercontent.com/359239/61795960-e52ef580-adfa-11e9-832f-bfbc1882fcfb.png">